Octokit.Tests.Conventions/Exception/InvalidUrlPropertyTypeException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Octokit.Tests.Conventions
+{
+    public class InvalidUrlPropertyTypeException : Exception
+    {
+        public InvalidUrlPropertyTypeException(Type modelType, IEnumerable<PropertyInfo> propertiesWithInvalidType)
+            : base(CreateMessage(modelType, propertiesWithInvalidType))
+        { }
+
+        static string CreateMessage(Type modelType, IEnumerable<PropertyInfo> propertiesWithInvalidType)
+        {
+            return string.Format("Model type '{0}' contains the following properties that are named or suffixed with 'Url' but which type is not String: {1}{2}",
```

Comment:
wording seems a little off here.  Maybe it could be "....  but are not of type String"